src/goInstallTools.ts

```diff
@@ -282,9 +282,38 @@ export function offerToInstallTools() {
 	}
 }
 
+function gopkgsMissing(): Promise<boolean> {
+	return new Promise<boolean>((resolve, reject) => {
+		const cmd = cp.spawn(getBinPath('gopkgs'), ['-help']);
+		cmd.stdout.on('data', (d) => {
+			resolve(true);
+		});
+
+		cmd.stderr.on('data', (d) => {
+			// expect the correct gopkgs, scan the output of the usage
+			const lines = d.toString().split('\n').filter((line) => (line.indexOf('Usage of') > -1) || (line.indexOf('custom output format') > -1) || (line.indexOf('Use -format to custom the output using template syntax') > -1));
+			resolve(lines.length !== 3);
+		});
+
+		cmd.on('error', (err) => {
+			if ((<any>err).code === 'ENOENT') {
+				return resolve(true);
+			}
+			reject(err);
+		});
+	});
+}
+
 function getMissingTools(goVersion: SemVersion): Promise<string[]> {
 	let keys = Object.keys(getTools(goVersion));
 	return Promise.all<string>(keys.map(tool => new Promise<string>((resolve, reject) => {
+		if (tool === 'gopkgs') {
+			gopkgsMissing().then((missing) => {
```

Comment:
This line will be reached only when there is no `gopkgs` tool in the user's machine.
Whereas you are expecting to reach here to make a decision on whether the `gopkgs` present in the user machine is the old one or the new one

Reply:
What do you think of this?
https://github.com/ramya-rao-a/vscode-go/commit/2119ec1849f18a0e2f9d5eae348a0d7c1e9cdd3f